prometheus_hardware_exporter/config.py

```diff
@@ -4,7 +4,7 @@
 from logging import getLogger
 from typing import List
 
-from pydantic import BaseModel, validator
+from pydantic import BaseModel, validator  # pylint: disable=no-name-in-module
```

Comment:
pylint complained:

```
prometheus_hardware_exporter/config.py:7:0: E0611: No name 'BaseModel' in module 'pydantic' (no-name-in-module)
```

Reply:
Do we consider to put the setting in pyproject.toml?
https://github.com/pydantic/pydantic/issues/1961

```
[tool.pylint.'MESSAGES CONTROL']
extension-pkg-whitelist = "pydantic"
```

I guess that is a tradeoff if we control pyproject.toml from automation.

Reply:
Ah I see, yes that could be useful to put in the pyproject.toml template, if we're using pydantic more often.